vocabs/DID-core-v1.jsonld

```diff
@@ -0,0 +1,159 @@
+{
+  "@context": {
+    "range": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#range",
+      "@type": "@id"
+    },
+    "comment": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#comment"
+    },
+    "isDefinedBy": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#isDefinedBy",
+      "@type": "@id"
+    },
+    "sameAs": {
+      "@id": "http://www.w3.org/2002/07/owl#sameAs",
+      "@type": "@id"
+    },
+    "domain": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#domain",
+      "@type": "@id"
+    },
+    "label": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#label"
+    },
+    "sec": "https://w3id.org/security#",
+    "as": "https://www.w3.org/ns/activitystreams",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "did": "https://www.w3.org/ns/did/vocab/v1#"
+  },
+  "@graph": [
+    {
+      "@id": "https://www.w3.org/ns/did/vocab/v1#",
+      "@type": "owl:Ontology"
+    },
+    {
+      "@id": "did:DIDSubject",
+      "@type": "rdfs:Class",
+      "comment": "Top level class for resources identified via a DID subject",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#core-properties",
+      "label": "DID Document"
+    },
+    {
+      "@id": "did:Service",
+      "@type": "rdfs:Class",
+      "comment": "Service endpoints are used in DID documents to express ways of communicating with the DID subject or associated entities",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#service-endpoints",
+      "label": "Service Endpoint"
+    },
+    {
+      "@id": "did:VerificationMethod",
+      "@type": "rdfs:Class",
+      "comment": "Superclass for all specific verification Methods. It is an 'abstract' class, never used by itself.",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#verification-methods",
+      "label": "Verification Method"
+    },
+    {
+      "@id": "did:alsoKnownAs",
+      "@type": "rdf:Property",
+      "comment": "This relationship is a statement that the subject of this identifier is also identified by one or more other identifiers.",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#also-known-as",
+      "sameAs": "as:alsoKnownAs"
+    },
+    {
+      "@id": "did:assertionMethod",
+      "@type": "rdf:Property",
+      "comment": "The assertionMethod property is used to express a verification relationship which indicates that a verification method can be used to verify a proof that a statement was asserted on behalf of the DID subject.",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#assertionmethod",
+      "range": "did:VerificationMethod",
+      "sameAs": "sec:assertionMethod"
```

Comment:
Now I'm confused... I don't think we should be duplicating classes w/ owl:sameAs... I do get the issue... I'm going to stop making suggestions from this point forward as we're probably not on the same page and need to get there... perhaps an interactive editing session might help.

Reply:
Yes, we should have a separate discussion on all this.

If we roll back the 'owl:sameAs' statements then:

- the DID vocabulary (RDFS, etc) in its own namespace consist of, from the top of my head, two terms, related to services. That is it.
- we could of course keep the current vocabulary files and keep everything to `sec:` but that looks quite strange: there is a namespace _for another_ vocabulary, whose proper definitions are defined in the namespace document for the `did:` vocabulary; moreover, the `sec:` vocabulary does not really have a proper definition. Messy.

We can do this of course, but I think it is ugly, and is really close to an anti-pattern for vocabulary definition. (I will hold my nose and not formally object, just to avoid confusion on that.)

Reply:
I think it's safe to say that none of us like where we are right now and ideally, we'd be depending on the Linked Data Security Vocabularies in a W3C namespace... but alas, we were blocked from starting there until there was demand for this stuff... and now that there is demand, we're having to do ugly things like this.

We'll just need to pick which ugly thing we think we can fix in time.

Reply:
@msporny o.k. I will roll back the `sameAs` statements, hold my nose, and get a cold beer from you when we meet next time personally (yeah, I know :-(

But I am still in the dark about the URL-s we should adopt for all these. Please, try to find some time going through the (tons of) comments about it because there are too many different branches now. See:

- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-759232149
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-765561004
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-766094311
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-766099819
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-766331323
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-770287530
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-770346191
- https://github.com/w3c/did-spec-registries/pull/183#issuecomment-770981036

There is now a total confusion :-)